is_core/menu.py

```diff
@@ -51,14 +57,28 @@ def get_menu_items(self, items):
                 if item.submenu_items:
                     item = deepcopy(item)
                     item.submenu_items = list(self.get_menu_items(item.submenu_items))
+
                 yield item
+
             else:
-                item = self.site._registry[item]
-                if isinstance(item, UIISCore):
```

Comment:
tento řádek tu byl úmyslně jelikož jen UIISCore umí vygenerovat odkaz, ostatní nemají metodu menu_url a to ti právě spadne už při generování atributů pro metodu get_menu_item tedy dříve než se to dostane k notimplementederror